tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py

```diff
@@ -37,7 +37,7 @@ def get_azure_core_credentials(**kwargs):
     # return ClientSecretCredential(
```

Comment:
Overall this change LGTM, but I'd also wonder if we want to put the import above for if this needs to be leveraged as well?  (Otherwise you'll have one additional layer of errors to work through in addition to just the azure-identity install)  I can rubber stamp this if we decide not.